irmaclient/client.go

```diff
@@ -683,6 +690,16 @@ func (client *Client) unenrolledSchemeManagers() []irma.SchemeManagerIdentifier
 	return list
 }
 
+func (client *Client) EnrolledSchemeManagers() []irma.SchemeManagerIdentifier {
+	list := []irma.SchemeManagerIdentifier{}
+	for name, manager := range client.Configuration.SchemeManagers {
+		if _, contains := client.keyshareServers[name]; manager.Distributed() && contains {
+			list = append(list, manager.Identifier())
+		}
+	}
+	return list
+}
```

Comment:
With the exception of an absent `!`, this function is code duplication from `client.unenrolledSchemeManagers()`. Simultaneously there are several inconsistencies between the two functions:
* This one is exported, the other one is not
* The purpose of the other one is to update state (namely the member `client.UnenrolledSchemeManagers`), while the output of this one is used directly in `irma_mobile`.

I think it would be best to solve the code duplication as well as these inconsistencies by:
* Adding a member `EnrolledSchemeManagers` to the `client` immediately below `UnenrolledSchemeManagers`
* Update (and rename) the `client.unenrolledSchemeManagers()` function such that it updates both the `client.EnrolledSchemeManagers` and `client.UnenrolledSchemeManagers` members
* in `irma_mobile`, use the new `client.EnrolledSchemeManagers` member instead of the method `client.EnrolledSchemeManagers()`.